src/Resources/config/autowire.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sonata\IntlBundle\Templating\Helper\DateTimeHelper" alias="sonata.intl.templating.helper.datetime" />
+        <service id="Sonata\IntlBundle\Templating\Helper\LocaleHelper" alias="sonata.intl.templating.helper.locale" />
+        <service id="Sonata\IntlBundle\Templating\Helper\NumberHelper" alias="sonata.intl.templating.helper.number" />
```

Comment:
Depending on the outcome of [this discussion](https://github.com/symfony/symfony-docs/pull/9886), I think it might be worth renaming the services, and doing the aliasing the other way around: `<service id="sonata.intl.templating.helper.datetime" alias="Sonata\IntlBundle\Templating\Helper\DateTimeHelper" />`.

I'm not quite sure the current best practice is really that good.

Reply:
@greg0ire IMO we could refactor this later across all bundles, but this PR looks "helpful" and fixes some DX problems for now.

Reply:
Ok. @XWB please fix the build